debian/rules

```diff
@@ -2,12 +2,15 @@
 
 export DEB_BUILD_OPTIONS := nocheck
 export DEB_DH_AUTORECONF_ARGS += --as-needed
+export DEB_BUILD_MAINT_OPTIONS = hardening=+all
 
 %:
-	dh $@ --with autoreconf,systemd
+	dh $@ --buildsystem=meson --with systemd
```

Comment:
Are we deprecating autotools? Just asking to see if there are plans to do so.

Reply:
According to the message added in the release 0.15.0:
```
configure.ac:41: warning: !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
configure.ac:42: warning: Autotools will be deprecated in the next Gstd release! Please migrate to Meson
configure.ac:43: warning: !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```
It seems 0.15.0 will be the last version with autotools support

Reply:
Nice